assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -116,69 +117,68 @@ export const TakeActionBoxoutEditor = ({
   const takeActionPageSelected = take_action_page && parseInt(take_action_page) > 0;
 
   if (loading || !actPageList.length) {
-    return __("Populating block's fields...", 'planet4-blocks-backend');
+    return __('Populating block\'s fields…', 'planet4-blocks-backend');
```

Comment:
I'm wondering how escaping characters will affect translations 🤔 Will they show as new strings? Will editors see the `\` in them and get confused?

Reply:
Minor issues, but better use 3 dots here at the end of the string, as it was originially, instead of a special character